test_cli.py

```diff
@@ -1053,6 +1053,26 @@ def test(ctx):
         [cl, 'run', 'cat %%%s//%s%%/stdout' % (source_worksheet_full, name)], expected_exit_code=1
     )
 
+    # Multi-alias tests
+    multi_alias_uuid1 = _run_command(
+        [cl, 'run', 'foo:{}'.format(uuid), 'foo1:{}'.format(uuid), 'echo "two alias"']
+    )
+    wait(multi_alias_uuid1)
+    check_equals('ready', _run_command([cl, 'info', '-f', 'state', multi_alias_uuid1]))
```

Comment:
I think `wait` returns whether it failed or not, so you don't need a separate `cl info` call.

Reply:
Yes. `Ready` is checked in `wait`. I removed all other similar cases as well in `test_cli.py`